cmd/doracled/cmd/register_oracle.go

```diff
@@ -117,7 +122,7 @@ func getTrustedBlockInfo(cmd *cobra.Command) (*panacea.TrustedBlockInfo, error)
 		return nil, fmt.Errorf("trusted block hash cannot be empty")
 	}
 
-	trustedBlockHash, err := base64.StdEncoding.DecodeString(trustedBlockHashStr)
+	trustedBlockHash, err := hex.DecodeString(trustedBlockHashStr)
```

Comment:
Oh, it seems that hex encoding/decoding is used for block hash, right?

Reply:
Right, As far as I know `blockHash` should use Hex.

Reply:
confirmed. Cool 👍